packages/internet-header/src/components/post-search/post-search.tsx

```diff
@@ -168,6 +169,13 @@ export class PostSearch implements HasDropdown, IsFocusable {
     }
     const query = this.searchBox.value.trim();
 
+    // shows or hides clearButton depending on the content of the searchbar
+    if (query !== '') {
+      this.clearButton?.classList.remove('visually-hidden');
```

Comment:
Toggling the `.visually-hidden` class may not be the best choice. In fact, this class is used to visually hide an element while leaving it accessible to screen readers.
This means that visually impaired users will still have access to the clear button when the search is empty.

To hide the button for everyone you can use the `hidden` property of HTMLElements: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden.